refactor(ivy): Move instructions back to ɵɵ

benlesh opened the pull request:
There is an encoding issue with using delta `Δ`, where the browser will attempt to detect the file encoding if the character set is not explicitly declared on a `<script/>` tag, and Chrome will find the `Δ` character and decide it is window-1252 encoding, which misinterprets the `Δ` character to be some other character that is not a valid JS identifier character

So back to the frog eyes we go.

```
    __
   /ɵɵ\
  ( -- ) - I am ineffable. I am forever.
 _/    \_
/  \  /  \
==  ==  ==
```

kara reviewed (changes requested):
Mostly looks good. Main issue is the public API guard looks off to me.

Also can we have a descriptive commit that doesn't say "frog eyes"?  e.g.

> refactor(ivy): switch back to ɵɵ prefix for instructions

benlesh commented on tools/public_api_guard/core/core.d.ts:
```diff
@@ -1051,383 +1051,3 @@ export interface WtfScopeFn {
-    providers: (Type<any> | ValueProvider | ExistingProvider | FactoryProvider | ConstructorProvider | StaticClassProvider | ClassProvider | any[])[];
-}
-
-export declare function Δinterpolation1(prefix: string, v0: any, suffix: string): string | NO_CHANGE;
```
Weird... I'll look and see what's going on.

benlesh replied:
Ah! I missed a regex in the ts_api_guardian_test config. Should be better now.

benlesh committed:
refactor(ivy): Move instructions back to ɵɵ

There is an encoding issue with using delta `Δ`, where the browser will attempt to detect the file encoding if the character set is not explicitly declared on a `<script/>` tag, and Chrome will find the `Δ` character and decide it is window-1252 encoding, which misinterprets the `Δ` character to be some other character that is not a valid JS identifier character

So back to the frog eyes we go.

```
    __
   /ɵɵ\
  ( -- ) - I am ineffable. I am forever.
 _/    \_
/  \  /  \
==  ==  ==
```

benlesh committed:
fixup! refactor(ivy): Move instructions back to ɵɵ

benlesh commented:
I'm really not sure why the bundle size went up so much. It shouldn't have changed much if at all. :thinking:

ocombe commented:
@benlesh it was an issue with the data in the database, restart circleCI and it should be fixed

JoostK commented on tools/ts-api-guardian/index.bzl:
```diff
@@ -24,7 +24,7 @@ def ts_api_guardian_test(
         actual,
         data = [],
-        strip_export_pattern = ["^__", "^ɵ"],
+        strip_export_pattern = ["^__", "^ɵ[^ɵ]"],
```
Faces everywhere!

benlesh commented:
[presubmit](http://test/OCL:249082951:BASE:249083024:1558375087745:e122595f)

kara reviewed (approved):
LGTM
